<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpLoadDialog</class>
 <widget class="QDialog" name="UpLoadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="sw_page">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>771</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="lb_notice">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>391</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>上传视频限制大小为50MB，格式flv或者mp4，
为了增加视频热度，上传时请勾选标签</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_path">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>491</width>
       <height>41</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_view">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>70</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_show">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>展示图</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_gif">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>611</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_process">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>72</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>上传进度</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="pgb_process">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>440</y>
       <width>531</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>16</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_startUpLoad">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>440</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>开始上传</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>150</y>
       <width>74</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cb_food">
        <property name="text">
         <string>美食</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_funny">
        <property name="text">
         <string>搞笑</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_energy">
        <property name="text">
         <string>正能量</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_dance">
        <property name="text">
         <string>舞蹈</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_music">
        <property name="text">
         <string>歌曲</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_video">
        <property name="text">
         <string>影视</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_outside">
        <property name="text">
         <string>户外</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_edu">
        <property name="text">
         <string>教育</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pb_UpLoad">
      <property name="text">
       <string>上传视频</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_videoRecording">
      <property name="text">
       <string>录制视频</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
